### PR TITLE
Danish translation updates

### PR DESCRIPTION
**Pull request by HThuren**
Summary of changes:
Companion.
   Many addons for new panels, labels, update and so on
   Possibility to change download katalog in opentxinterface.cpp by translation. This also gave minor change to other
   translation files.

Radio
   Changes at da.h according to labels and minor changes

Bugs.
   This PR may wait for solution to bug in PR #2324

### CODE REVIEW

**Comment by pfeerick** [comment]
Crossreference: The bug alluded to above for the merged PR#2324 was mentioned https://github.com/EdgeTX/edgetx/pull/2324#issuecomment-1281135440

**Comment by elecpower** [comment]
I am not surprised by the bug but was hoping it would not surface. I have a fix in progress as part of a larger tidy up but I will extract it into a discreet PR.

**Comment by HThuren** [comment]
> I am not surprised by the bug but was hoping it would not surface. I have a fix in progress as part of a larger tidy up but I will extract it into a discreet PR.

I think, its best to merge then, and I translate the rest, when the bug are done (and maybe other stuff :-)

**Comment by HThuren** [comment]
Just some small corrections to radio

File: companion/src/firmwares/opentx/opentxinterface.cpp [hunk 1]
```diff
@@ -1217,7 +1217,7 @@ unsigned long OpenTxEepromInterface::loadBackup(RadioData &radioData, const uint
 
 QString OpenTxFirmware::getReleaseNotesUrl()
 {
-  return QString("%1/downloads").arg(EDGETX_HOME_PAGE_URL);
+  return QString( tr("%1/downloads")).arg(EDGETX_HOME_PAGE_URL);
```
Comment (pfeerick) [inline companion/src/firmwares/opentx/opentxinterface.cpp:1220]: Er... is it valid to be translating this? Isn't it constructing the release notes URL path? i.e. `EDGETX_HOME_PAGE_URL/downloads` (which isn't actually used/valid anyway - it directly goes to QString link = "https://github.com/EdgeTX/edgetx/releases";

Comment (HThuren) [reply companion/src/firmwares/opentx/opentxinterface.cpp:1220]: yrk,. was supposed to change the local catalog at $USER/EdgeTx/downloads.
I do a rollback, but the rollback will not give **"https://github.com/EdgeTX/edgetx/releases"** but (in constant.h) **"https://edgetx.org"**
I'll change to ..edgetx/releases in the translation files , ok ?

Comment (HThuren) [reply companion/src/firmwares/opentx/opentxinterface.cpp:1220]: no, will not work, damn, I restore the translation files but danish one, and change constant.h to the new one, ok ?

Comment (pfeerick) [reply companion/src/firmwares/opentx/opentxinterface.cpp:1220]: You should be able to just undo all the changes to everything but Danish translation file... if needed I can probably roll that back, just me know if you give up ;)

The change you made here `opentxinterface.cpp` was the addition of the `tr()` - I was just pointing that string isn't even used, even it it was valid to be translated...
image

Comment (HThuren) [reply companion/src/firmwares/opentx/opentxinterface.cpp:1220]: I try to make the rollback, something to learn :-)

**Pull request closed without merging**